### PR TITLE
WebRTC client side

### PR DESCRIPTION
**Pull request by enahum**
#### Summary
This is the client side for WebRTC (EE)

#### Ticket Link
https://mattermost.atlassian.net/browse/PLT-3412
https://mattermost.atlassian.net/browse/PLT-3943

#### Depends on:
https://github.com/mattermost/platform/pull/3706

#### Checklist
[Place an '[x]' (no spaces) in all applicable fields. Please remove unrelated fields.]
- [ ] Added or updated unit tests (required for all new features)
- [ ] Added API documentation (required for all new APIs)
- [x] All new/modified APIs include changes to the drivers
- [x] Has enterprise changes (https://github.com/mattermost/enterprise/pull/49)
- [x] Has UI changes
- [x] Includes text changes and localization files updated
- [ ] Touches critical sections of the codebase (auth, upgrade, etc.)

@jasonblais  @asaadmahmood please do the UI Tweaks agains this branch

### CODE REVIEW

**Comment by enahum** [comment]
PR failing cause dependency of #3706

**Comment by crspeller** [comment]
@enahum It should detect a branch with the same name in the enterprise repository as long as you push it first. I restarted to see if that will work.

**Comment by crspeller** [comment]
Wait.. what's up with your branch names? Is this just a merge into a feature branch? If so it should have the same name in both repositories. Can you fix the branch names please?

**Comment by enahum** [comment]
@crspeller actually no need for this PR to build, this is a PR against branch PLT-3412, which has the same name on `platform` and `enterprise`.

I have fired up a spinmint manually.

**Comment by coreyhulen** [comment]
+1

**Comment by jasonblais** [comment]
Elias: For next week, can you help set up a manual test server for us to try the latest changes out?

**Comment by enahum** [comment]
@jasonblais sure thing, I'll update the current webrtc spinmint with the latest changes

**Comment by jasonblais** [comment]
Update: Tested and will be reviewed with @enahum on Monday, after release work is done for v3.3

Close to being ready, excited!

File: webapp/components/webrtc/components/webrtc_notification.jsx [hunk 1]
```diff
@@ -0,0 +1,319 @@
...
+                    <div>
+                        {profileImg}
+                    </div>
+                    {msg}
+                </div>
+            );
+        } else if (this.state.rhsOpened && WebrtcStore.isBusy()) {
+            return (
+                <div
+                    className='webrtc-notification__rhs'
```
Comment (enahum) [inline webapp/components/webrtc/components/webrtc_notification.jsx]: @asaadmahmood check this out, I created this class so you can style the ongoing call indicator

**Comment by jasonblais** [comment]
Latest updates looks good, tested with various people.

Nothing that should block merging behind the preview feature flag. Remaining open items triaged with @enahum.

**Comment by coreyhulen** [comment]
+1

**Comment by crspeller** [comment]
@enahum Can you fix the build issues so we can get this in?

**Comment by enahum** [comment]
closing and resubmitting against master instead of PLT-3412 branch

**Pull request closed without merging**